src/codegen/utilities/interpreter_helpers.py

```diff
@@ -64,14 +64,12 @@
     "GetSyncPulseTimeWhen",
     "GetTimingAttributeExTimestamp",
     "GetTimingAttributeTimestamp",
-    "GetTrigAttributeTimestamp",
     "SetArmStartTrigTrigWhen",
     "SetFirstSampClkWhen",
     "SetStartTrigTrigWhen",
     "SetSyncPulseTimeWhen",
     "SetTimingAttributeExTimestamp",
     "SetTimingAttributeTimestamp",
```

Comment:
FYI, you will need these when you add the rest of the attributes, since first_samp_timestamp_val and sync_pulse_when are timing attributes.

Reply:
Noted with thanks, I will add support for all of these functions at one shot in another PR